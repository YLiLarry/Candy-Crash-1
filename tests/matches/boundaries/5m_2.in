5 ..
swap 3 2 down
