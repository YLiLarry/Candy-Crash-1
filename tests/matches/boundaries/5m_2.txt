__1 __2 __3 __0 __1
__2 __3 __0 __1 __2
__3 __0 __1 __2 __3
__0 __2 __1 __2 __0
__1 __1 __0 __1 __1
0000000000