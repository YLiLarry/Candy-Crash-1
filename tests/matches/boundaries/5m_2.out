
__1 __2 __3 __0 __1 
__2 __3 __0 __1 __2 
__3 __0 __1 __2 __3 
__0 __2 __1 __2 __0 
__1 __1 __0 __1 __1 

__1 __2 __3 __0 __1 
__2 __3 __0 __1 __2 
__3 __0 __1 __2 __3 
__0 __2 __0 __2 __0 
__1 __1 __1 __1 __1 

__1 __2 __3 __0 __1 
__2 __3 __0 __1 __2 
__3 __0 __1 __2 __3 
__0 __2 __0 __2 __0 
___ ___ _p1 ___ ___ 

__0 __0 __3 __0 __0 
__1 __2 __0 __0 __1 
__2 __3 __1 __1 __2 
__3 __0 __0 __2 __3 
__0 __2 _p1 __2 __0 
